src/components/GroupBuildsList/GroupBuildsList.tsx

```diff
@@ -0,0 +1,138 @@
+import {
+  DescriptionList,
+  DescriptionListDescription,
+  DescriptionListGroup,
+  DescriptionListTerm,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import { TableComposable, Tbody, Td, Th, Thead, Tr } from '@patternfly/react-table';
+import { useMemo, useState } from 'react';
+
+import { GroupBuild } from 'pnc-api-types-ts';
+
+import { PageTitles } from 'common/constants';
+import { getFilterOptions, getSortOptions } from 'common/entityAttributes';
+import { groupBuildEntityAttributes } from 'common/groupBuildEntityAttributes';
+
+import { IServiceContainer } from 'hooks/useServiceContainer';
+import { ISortOptions, useSorting } from 'hooks/useSorting';
+
+import { BuildName } from 'components/BuildName/BuildName';
+import { BuildStatusIcon } from 'components/BuildStatusIcon/BuildStatusIcon';
+import { ContentBox } from 'components/ContentBox/ContentBox';
+import { Filtering } from 'components/Filtering/Filtering';
+import { Pagination } from 'components/Pagination/Pagination';
+import { ServiceContainerLoading } from 'components/ServiceContainers/ServiceContainerLoading';
+import { SortGroup } from 'components/SortGroup/SortGroup';
+import { Toolbar } from 'components/Toolbar/Toolbar';
+import { Username } from 'components/Username/Username';
+
+import { areDatesEqual, calculateDuration, createDateTime } from 'utils/utils';
+
+interface IGroupBuildsListProps {
+  serviceContainerGroupBuilds: IServiceContainer;
+  componentId: string;
+}
+
```

Comment:
I would suggest also implementing `defaultColumns` feature since columns would be different on other pages(like the homepage dashboards that have no users column)

Reply:
done